.env.testing

```diff
@@ -1,5 +1,5 @@
 ES_URL=http://localhost:19200
-NEO4J_URL=bolt://localhost:17687
+NEO4J_URL=bolt://xd:dd@localhost:17687
```

Comment:
It gets even worse :D whats that?

Reply:
Good catch. I didn't know if leaving empty worked so I added random shit here. gonna remove that :D